tests/components/sma/__init__.py

```diff
@@ -110,30 +110,23 @@
     original_name="pv_power",
 )
 
+MOCK_CONFIG_ENTRY = MockConfigEntry(
```

Comment:
It's better to use a pytest fixture that returns a MockConfigEntry. The MockConfigEntry instance will change it's state in the tests, and we want a clean instance in each test, so it should not be a constant like this.

Reply:
I've moved it to fixture in a new `conftest.py` file.
https://github.com/home-assistant/core/pull/49346/commits/e77d649af20cf660655f4af697702a69d8ef0e05